presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -193,6 +193,13 @@ protected RowExpression visitExpression(Expression node, Void context)
             throw new UnsupportedOperationException("not yet implemented: expression translator for " + node.getClass().getName());
         }
 
+        @Override
```

Comment:
What is this change about?

Reply:
This is an annoying corner case from #9711: `identity` should never exist in a `RowExpression` with the exception of a constant lambda like `VALUES transform(sequence(1, 5), x -> x * x)`, where `x` is an identifier but actually has values. #9711 resolved this in expression interpreter. We might need to fix it in translator and row expression interpreter